src/content/release-notes.mdx

```diff
@@ -5,6 +5,46 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.36
+
+5 September 2025
+
+This version is compatible with Kubernetes versions 1.31 to 1.33
+Okteto Chart release 1.36 is designed to work with [Okteto CLI 3.11.x](https://github.com/okteto/okteto/releases/tag/3.11.0)
+
+### [AI Agent Fleets Now Available (Beta)](ai-agent-fleets/index.mdx)
+AI Agent Fleets brings the power of AI directly into your development workflow, powered by Claude Code from Anthropic. Each agent operates independently in its own Kubernetes namespace with full isolation, giving you the safety and control you need for AI-assisted development.
+
+#### Key Capabilities
+
+- **Intelligent Repository Onboarding**: Agents can quickly understand and work with existing codebases, analyzing project structure, dependencies, and patterns to get up to speed faster than ever
+- **Application Scaffolding**: Generate new services, APIs, and applications from natural language descriptions, complete with best practices and proper project structure
+- **Task Automation**: Automate common development tasks like adding endpoints, refactoring code, updating dependencies, or implementing new features
+- **Real Containerized Environments**: All agents run in production-like Kubernetes environments with access to the same runtime, secrets, and configurations as your actual development setup
+- **Parallel Execution**: Launch multiple agents simultaneously to work on different features or experiments independently
+
+#### For Administrators
+Organization administrators have full control over AI Agent Fleets deployment:
+- Enable the feature for your organization through the Admin Dashboard under Admin > AI Agent Fleets
+- Toggle access per user to control who can launch AI agents in your organization
+- Configure LLM keys - Provide your own Anthropic API key (directly from Anthropic or via Amazon Bedrock)
+
+We're continuously improving AI Agent Fleets based on your feedback. Try it today and let us know how it transforms your development workflow!
+
+### New Features {#new-features-1.36}
+
+- Added a [new Public API endpoint to delete users](admin/okteto-api.mdx)  <!-- 9572 -->
+- Improved installer reliability by adding a preflight init container  <!-- 9598 -->
+
+### Improvements {#improvements-1.36}
+
+- Pre-pull Okteto images used by jobs to accelerate job start times  <!-- 9601 -->
+- Installer now retries transient connection issues  <!-- 9608 -->
+
+### Bug Fixes {#bug-fixes-1.36}
+
+- Fixed previews layout issue that could hide logs when breadcrumb is visible  <!-- 9588 -->
```

Comment:
Fixes to add related to CLI:

- [Okteto CLI 3.11.0](https://github.com/okteto/okteto/releases/tag/3.11.0): Fixed a panic when deploying an Okteto manifest with divert through `okteto up` or `okteto test` tests
- [Okteto CLI 3.11.0](https://github.com/okteto/okteto/releases/tag/3.11.0): Fixed a problem with the flag `--wait` in the `okteto deploy` command when deploying a subset of services from a compose. The command was being stuck until the timeout was being triggered